src/Maestro/maestro-angular/src/app/page/build/build.component.ts

```diff
@@ -210,24 +214,49 @@ export class BuildComponent implements OnInit, OnChanges {
             return true;
           }
           // emit only the first "Loading" instance so refreshes don't cause the loading spinner to show up
-          if (!emittedLoading)  {
+          if (!emittedLoading) {
             emittedLoading = true;
             return true;
           }
           return false;
         }),
       ),
     );
+
+    this.subscriptionsList$ = this.build$.pipe(
```

Comment:
We don't seem to use the includeSubToolsets variable when grabbing the list of subscriptions. Is that future work? Should we just include it in the original design? Does having a toolset toggle even make sense for subscriptions?

Reply:
Yes, it's future work if it's necessary - that information comes from either a very complicated build graph query (which would require that I pull in the build graph to this component) or it comes from VersionDetails, which I'm in the process of loading anyway. The PKPIs split product vs toolset, so I assumed it was an important part of the metric but it's good to double check.

Reply:
Moving the toolsets question out into a separate issue/PR so this no longer applies.